assembly-combined-package/assembly-combined/sbin/linkis-start-all.sh

```diff
@@ -90,6 +90,35 @@ SERVER_NAME="ps-cs"
 SERVER_IP=$CS_INSTALL_IP
 startApp
 
+#datasourcemanager
+SERVER_NAME="ps-datasourcemanager"
+SERVER_IP=$DATASOURCEMANAGER_INSTALL_IP
+startApp
+
+#metadatamanager
+SERVER_NAME="ps-metadatamanager"
+SERVER_IP=$METADATAMANAGER_INSTALL_IP
+startApp
+
+#metadatamanager-es
+SERVER_NAME="ps-metadatamanager-es"
+SERVER_IP=$METADATAMANAGER_ES_INSTALL_IP
+startApp
+
+#metadatamanager-hive
+SERVER_NAME="ps-metadatamanager-hive"
+SERVER_IP=$METADATAMANAGER_HIVE_INSTALL_IP
+startApp
+
+#metadatamanager-kafka
+SERVER_NAME="ps-metadatamanager-kafka"
+SERVER_IP=$METADATAMANAGER_KAFKA_INSTALL_IP
+startApp
```

Comment:
So many DataSource process, I suggest create a new start DataSources script to start it.

Reply:
thanks，I will fix it later